lib/widgets/resources/board.dart

```diff
@@ -11,10 +13,12 @@ import '../button.dart';
 class ViamBoardWidget extends StatefulWidget {
   /// The [Board]
   final Board board;
+  final Duration refreshInterval;
```

Comment:
Add a docstring comment describing this property please

Reply:
Added